pkg/controllers/namespace/namespace_controller_test.go

```diff
@@ -121,111 +109,3 @@ func TestInsertNamespaceNetworkconfigBinding(t *testing.T) {
 		})
 	}
 }
-
-func TestCreateVPCCR(t *testing.T) {
```

Comment:
we need to add the unit test back when we have time...

Reply:
ok